bfh/version.py

```diff
@@ -1 +1 @@
-__version__ = "0.2.0"
+__version__ = "0.2.1"
```

Comment:
semver here, new feature that doesn't break compatability gets a minor version, so this will be 0.4.0